setup.py

```diff
@@ -37,6 +37,7 @@ def find_version(*file_paths):
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
```

Comment:
I wonder if it is worth widening this PR to also add Django 4.0 support (add to test suite as well).